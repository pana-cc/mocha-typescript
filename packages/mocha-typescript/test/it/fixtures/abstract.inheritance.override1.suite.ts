import { assert } from "chai";
import { suite, test } from "../../../index";

abstract class AbstractTestBase {

  @test public "inherited test from AbstractTestBase"() {

    assert.fail("derived suites must implement this");
  }
}

@suite class ConcreteTest extends AbstractTestBase {

}
