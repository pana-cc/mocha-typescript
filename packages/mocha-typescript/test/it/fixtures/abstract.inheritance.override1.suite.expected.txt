
  ConcreteTest
    1) inherited test from AbstractTestBase
  0 passing
  1 failing
  1) ConcreteTest
       inherited test from AbstractTestBase:
     AssertionError: derived suites must implement this
